app/ZenNode.js

```diff
@@ -157,10 +159,17 @@ class ZenNode {
     if (net) {
       args.push('--chain', net)
     }
-    if (process.env.ZEN_NODE_API_PORT) {
-      args.push('--api', `127.0.0.1:${process.env.ZEN_NODE_API_PORT}`)
+    if (isWindows()) {
+      args.push('--api', `localhost:${getPort()}`)
     }
+    if (process.env.ZEN_NODE_API_PORT) {
 
+      if (isWindows()) {
+        args.push('--api', `localhost:${process.env.ZEN_NODE_API_PORT}`)
```

Comment:
if user is on windows and `process.env.ZEN_NODE_API_PORT` is defined, we'll be passing two `--api` flags to the node, is that okay? "lest flag wins"?

Reply:
it is fine. I also handle that case in use localhost instead of 127.0.0.1

Reply:
Sorry, we don't actually want that. I will fix